src/epub/create_epub.py

```diff
@@ -0,0 +1,165 @@
+import os
+import zipfile
+
+from lxml import etree
+import shutil
+from zipfile import ZipFile
+from datetime import datetime
+
+import src.html.create_songs_html as cash
+
+
+class SongMeta:
+    def __init__(self, title='', alias='', plik=''):
+        self.title = title if title else ''
+        self.alias = alias if alias else ''
+        self.plik = plik
+
+    # def __repr__(self):
+    #     return "(%s, alias: %s, plik: %s)" % (self.title, self.alias, self.plik)
+
+    @staticmethod
+    def parseDOM(root, plik):
+        def elementTextOrNone(elem):
+            return elem.text if elem is not None else None
+        return SongMeta(
+            title=root.get('title'),
+            alias=elementTextOrNone(root.find('{*}alias')),
+            plik=plik
+        )
+
+
+def add_song(path, lista):
+    tree = etree.parse("../../songs/" + path + ".xml")
+    plik = path + ".xml"
+    song = SongMeta.parseDOM(tree.getroot(), plik)
+    lista.append(song)
+
+
+def list_of_song(path_out):
+    songs_list = os.listdir(path_out)
+    list_od_meta = []
+    for song in songs_list:
+        if song[-5:] == '.html':
```

Comment:
There is `startswith` on string... 
I believe there is also endswith that can be used in that context.

Reply:
But usage of: 

https://docs.python.org/3/library/os.path.html#os.path.splitext would be even better.

Reply:
done